File: app/src/main/java/com/mgaetan89/showsrage/fragment/SettingsAboutFragment.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

public class SettingsAboutFragment extends SettingsFragment {
	@Override
	protected int getTitleResourceId() {
		return R.string.about;
	}

	@Override
	protected int getXmlResourceFile() {
		return R.xml.settings_about;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutFragment.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R

open class SettingsAboutFragment : SettingsFragment() {
	override fun getTitleResourceId() = R.string.about

	override fun getXmlResourceFile() = R.xml.settings_about

	companion object {
		fun newInstance() = SettingsAboutFragment()
	}
}
